src/core/keybindings.c

```diff
@@ -2846,8 +2846,8 @@ process_workspace_switch_grab (MetaDisplay *display,
                                                   window,
                                                   event->xkey.time);
             }
-            
-          
+
```

Comment:
I think you can remove the double line too, when you already touch this file, or not?